starters/features/storybook/.storybook/main.ts

```diff
@@ -0,0 +1,11 @@
+import type { StorybookConfig } from '@storybook/builder-vite';
+
+const config: StorybookConfig = {
+  addons: ['@storybook/addon-essentials'],
+  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
```

Comment:
The storybook cli scaffolds `*.mdx` for all frameworks (not `*.stories.mdx`). I'm guessing this is because `stories.mdx` is a special pattern that will make the file declare stories rather than just be documentation.
Picking up all mdx files can cause conflicts with Qwik-city mdx though, so having _some_ explicit indicator that mdx is for storybook, not qwik city is important (because Qwik-city mdx will only work with Qwik component, and SB mdx will only work with react components).
I think a "storybook" prefix gets the explicit marking, without triggering the special behavior of `stories.mdx`.
```suggestion
  stories: ['../src/**/*.storybook.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
```
But, because of some limitations, I'm planning on just doing `*.mdx` in the storybook CLI qwik integration like other frameworks. I'll need to add documentation to warn about the Qwik-city conflict potential. You could do that if you want to match how the sb cli will eventually work (based on my current guess).

Reply:
in a `stories.mdx` file, any story you reference will get added to the navigation. If you already declared the story in a `stories.tsx` file, this will cause duplicates.
with any other pattern, you can reference the stories in the mdx without adding them to the side bar.
I suppose it could go either way, but I defer to the default in [storybook CLI](https://github.com/storybookjs/storybook/blob/next/code/lib/cli/src/generators/configure.ts#L31)